app/src/main/java/org/fossasia/phimpme/utilities/ImgurApi.java

```diff
@@ -0,0 +1,15 @@
+package org.fossasia.phimpme.utilities;
+
+import android.util.ArrayMap;
+import com.google.gson.JsonElement;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.Header;
+import retrofit2.http.POST;
+
+public interface ImgurApi {
+
+  @POST("/image")
+  Call<JsonElement> uploadImageToImgur(
```

Comment:
JsonElement to as well

Reply:
? Response to a class? why?

Reply:
Do we have a class for every API response? why should be follow this approach?

Reply:
See the extension of the file

Reply:
i will change but JsonElement is also a class present under  ```package com.google.gson;```

Reply:
Obviously, it is. You can even use ArrayList without type parameters to make it like Python and JS. You can do anything in Java that you can do in Python or JS, but we want [type safety](https://en.wikipedia.org/wiki/Type_safety)